src/vm/wren_compiler.c

```diff
@@ -3364,6 +3381,20 @@ static void import(Compiler* compiler)
   } while (match(compiler, TOKEN_COMMA));
 }
 
+// Consume the optional type definition.
+static void optionalTypeDefinition(Compiler* compiler)
+{
+  if (match(compiler, TOKEN_COLON))
+  {
+    ignoreNewlines(compiler);
+    parsePrecedence(compiler, PREC_IS);
+    
+    // Discard the type definition.
+    // This compiler is too simple to handle optional type definition.
+    emitOp(compiler, CODE_POP);
```

Comment:
@mhermier if the type definition is entirely ignored, why does it affect the bytecode with a pop here? (I'm not reading too closely!)